tools/cmake/cmbuild

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+# make like wrapper around "cmake --build"
+# (c) 2020 Axel Kohlmeyer <akohlmey@temple.edu>
+# This file is in the public domain
+
+WORKDIR="${PWD}"
+PARAMS=""
+MYARGS=""
+
+usage()
+{
+   echo "Usage: cmbuild [-v] [-h] [-C <dir>] [-j <num>] [<target1> [<target2>] ...]" >&2
+}
+
+help()
+{
+   usage
+   cat >&2 <<EOF
+
+Options:
+  -h                print this message
+  -j <NUM>          allow processing of NUM concurrent tasks
+  -C DIRECTORY      execute build in folder DIRECTORY
+  -v                produce verbose output
+EOF
+}
+
+while (( "$#" )); do
+    case "$1" in
+      -C)
+        WORKDIR="$2"
+        shift 2
+        ;;
+      -v)
+        MYARGS="${MYARGS} -v"
+        shift
+        ;;
+      -h)
+        help
+        exit 2
+        ;;
+      -j)
+        MYARGS="${MYARGS} -j $2"
+        shift 2
+        ;;
+      --)
+        shift
+        break
+        ;;
+      -*)
+        echo "Error: Unsupported flag $1" >&2
+        echo
+        usage
+        exit 1
+        ;;
+      *)
+        PARAMS="${PARAMS} $1"
+        shift
+        ;;
+    esac
+done
+
+if [ ! -f "${WORKDIR}/CMakeCache.txt" ] ; then
+   echo "Must execute in a CMake build directory or use -C flag to select one" >&2
+   exit 3
+fi
+
+eval set -- "${PARAMS} $@"
+exec cmake --build "${WORKDIR}" ${MYARGS} --target "$@"
```

Comment:
I not 100% if this script is really necessary if it runs a single command....

Reply:
it is a single command with a horribly complex and difficult to remember syntax and requirements. `cmbuild` (i am open for other names) is so much easier to remember and can be used like `make clean`, `make lmp`, `make` and does about the same thing. also, it is much less effort to type `-v` than VERBOSE=1.

Reply:
I thought about this again and one of the main points for the cmake buildsystem is to make things more standard! Currently `cmbuild` doesn't do much, but it is the start down the dark path of custom scripts, e.g. `make.py`, all over again.

At this point, if you think `cmake --build .` is too much of a monster, I would say let's just stick to `make` in the documentation and don't add `cmbuild`. 95% of users won't use anything but the defaut (Unix Makefiles) anyhow, so let's either teach them the cmake standard way or leave it alone.